### PR TITLE
Add shikiji twoslash

### PR DESCRIPTION
**Pull request by jellli**
Add twoslash support https://github.com/shuding/nextra/issues/2525

### CODE REVIEW

**Review by dimaMachina — COMMENTED** [review]
I will rebase your PR on top of https://github.com/shuding/nextra/pull/2679

File: .changeset/rich-apricots-bake.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'nextra-theme-blog': minor
+'nextra-theme-docs': minor
+'nextra': minor
+---
+
+add shikiji twoslash
```
Comment (dimaMachina) [inline .changeset/rich-apricots-bake.md:7]:
```suggestion
add shikiji twoslash

Demo feature: https://nextra-v2-na3obnhub-shuding1.vercel.app/docs/guide/twoslash-support
```

**Review by dimaMachina — APPROVED** [review]
thank you!

**Comment by Jack-Works** [comment]
hi @jellli I tried this but it does not work for an unknown reason. also the package has been renamed to `@shikijs/twoslash`

**Comment by jellli** [comment]
> hi @jellli I tried this but it does not work for an unknown reason. also the package has been renamed to `@shikijs/twoslash`

Hi, which version are you currently using? Thanks for letting me know the rename thing, I will check that out.

**Comment by Jack-Works** [comment]
> Hi, which version are you currently using

I'm using 3.0.0-alpha.17

**Pull request merged**